<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsForm</class>
 <widget class="QDialog" name="OptionsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>210</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>168</width>
     <height>95</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="gameSettings">
      <property name="minimumSize">
       <size>
        <width>144</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Игрульки</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="programSetting">
      <property name="minimumSize">
       <size>
        <width>144</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Програмульки</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>-30</y>
     <width>41</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoStartMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Автозапуск</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="fullscreenMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>На весь экран</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="buttonIconMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Иконки кнопок</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="defaultBackgroundMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Стандартные обои</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="wallpaperPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Фоновое изображение</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Начальная директория</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>28</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>28</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxStyles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
